src/CMakeLists.txt

```diff
@@ -3,24 +3,52 @@ include_directories(${PROJECT_SOURCE_DIR}/headers/algorithms)
 include_directories(${PROJECT_SOURCE_DIR}/headers/python_bindings)
 include_directories(${PROJECT_SOURCE_DIR}/headers/carma/include)
 include_directories(${PROJECT_SOURCE_DIR}/headers/carma/include/carma_bits)
+# Include directory for windows
+if(WIN32)
+include_directories(${PROJECT_SOURCE_DIR}/headers/armadillo/include)
+include_directories(
+        ${PROJECT_SOURCE_DIR}/headers/armadillo/include/armadillo_bits)
+set(OPENBLAS_dir ${PROJECT_SOURCE_DIR}/headers/armadillo/examples/lib_win64)
```

Comment:
According to `cmake-lint` (which I ran using `cmake-lint src/CMakeLists.txt`), we should not indent inside `if` blocks. Please let me know if you think otherwise.